UniversityApp/DTOs/StudentSearchParametersDto.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace UniversityApp.DTOs
+{
+    public class StudentSearchParametersDto
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int? Age { get; set; }
+        public string Gender { get; set; }
+
+        [Required()]
+        public int PageNumber { get; set; }
+        [Required()]
+        [Range(Constants.MIN_PAGE_SIZE, Constants.MAX_PAGE_SIZE)]
```

Comment:
Please use for it fluent validation in next step.

Think how could you test validation comparing attributes and using fluentvalidation